CMakeLists.txt

```diff
@@ -44,7 +44,22 @@ if(BUILD_MCNP5 OR BUILD_MCNP6)
   enable_language(Fortran)
 endif()
 
-find_package(MOAB REQUIRED)
+# Dealing with MOAB
+if (PULL_INSTALL_MOAB)
+  set(MOAB_VERSION ${PULL_INSTALL_MOAB})  
+
+  # Ensure MOAB_VERSION
+  if(${MOAB_VERSION})
```

Comment:
It's not clear to me what conditions re true for this?  It seems like it would evaluate to true if I *DID* set the version? but maybe I don't know how this CMake evaluates `if(5.5.3)`, for example?

Reply:
```
if(<variable>)
True if given a variable that is defined to a value that is not a false constant. False otherwise, including if the variable is undefined. Note that macro arguments are not variables.
``` from https://cmake.org/cmake/help/latest/command/if.html

Reply:
Is a version string evaluated as a "False" constant then? I would have assumed that a version string would evaluate as `True` and cause this message and default to happen??

Reply:
you are right I miss understood this. I'll fix it!

Reply:
@gonuke this should take care of it !